include/platform-aws.h

```diff
@@ -72,7 +72,15 @@ class PlatformAWS : public Platform {
 		return fields ? fields->func_idx : -EIO;
 	}
 
+	// Determine AWS at runtime
+	bool is_aws() const;
+
 private:
+	// Constants
+	static constexpr int BASE_PRIORITY = 100;
+	static constexpr unsigned short AWS_VENDOR_ID = 0x1D0F;
+	static constexpr unsigned short EFA_DEV = 0xEFA0;
```

Comment:
This isn't right.  I'd just have a function `is_efa_device()` that has 0xEFA0 and the 0xFFF mask hard coded, with comments.  Because this method just adds confusion.